cmd/kola/kola.go

```diff
@@ -310,6 +315,21 @@ func runList(cmd *cobra.Command, args []string) {
 		fmt.Fprintln(w, "Test Name\tPlatforms\tArchitectures\tDistributions")
 		fmt.Fprintln(w, "\t")
 		for _, item := range testlist {
+			platforms := strings.Join(item.Platforms, " ")
```

Comment:
Couldn't all of this be solved with two loops through `item.Platforms` & `item.Distros`? i.e.
```golang
for _, platform := range item.Platforms {
    if platform == "all" || platform == listPlatform {
        break
    }
}
for _, distro := range item.Distros {
    if distro == "all" || distro == listDistro {
        break
    }
}
fmt.Fprintf(w, "%v\n", item)
```

Reply:
Thank you for reviewing!
Initially I was trying to avoid using for loops so ended up with this awkward readability code. Will add a commit with the above changes.